packages/office-addin-debugging/test/test.ts

```diff
@@ -76,3 +80,50 @@ describe("port functions", function() {
   });
 
 });
+
+describe("start/stop functions", function() {
+  const pid = 1234;
+  it("writing process id file", async function() {
+    await debugInfo.saveDevServerProcessId(pid);
+    const json = fs.readFileSync(debugInfo.getDebuggingInfoPath());
+    const devServerInfo = JSON.parse(json.toString());
+    const processId = devServerInfo.devServer.processId;
+    assert.strictEqual(processId.toString(), pid.toString());
+  });
+  it("reading process id file", async function() {
+    const id = debugInfo.readDevServerProcessId();
+    if (id) {
+      assert.strictEqual(id.toString(), pid.toString());
+    }
+  });
+  it("deleting process id file", async function() {
+    debugInfo.clearDevServerProcessId();
+  });
+  it("read process id file that is missing", async function() {
+    const id = debugInfo.readDevServerProcessId();
+    assert.strictEqual(id, undefined);
+  });
+  it("write process id file that already exists", async function() {
+    const secondPid = 5678;
+    await debugInfo.saveDevServerProcessId(pid);
+    await debugInfo.saveDevServerProcessId(secondPid);
+    const json = fs.readFileSync(debugInfo.getDebuggingInfoPath());
+    const devServerInfo = JSON.parse(json.toString());
+    const processId = devServerInfo.devServer.processId;
+    assert.strictEqual(processId.toString(), secondPid.toString());
```

Comment:
do you need to use toString() here if they are both numbers?

Reply:
Fixed. toString() is not needed.